assets/css/fw.css

```diff
@@ -331,6 +335,10 @@ main>div {
     color: #3daed2;
   }
 
+  .blue {
+    color: #3daed2!important;
+  }
+
```

Comment:
`.blue` being used for two different colors sounds a bit confusing :)

Reply:
That’s intended: one is for the dark mode while the other one is for the normal white mode. That’s css magic ;-).

Reply:
> That’s intended: one is for the dark mode while the other one is for the normal white mode. That’s css magic ;-).

Should have read more context... :)